scripts/run_tests.py

```diff
@@ -452,6 +452,45 @@
     "exit",
 ]
 
+cwl_cromwell_tests_list = [
+    "cwl_ad_hoc_file_test",
+    "cwl_cache_between_workflows",
+    "cwl_cache_within_workflow",
+    "cwl_docker_size",
+    "cwl_dynamic_initial_workdir",
+    "cwl_expressionLib",
+    "cwl_format",
+    # "cwl_format_url", # Could not load extension schema https
```

Comment:
For ignored tests, please reference the Jira id where the issue will be resolved, so we know when to un-ignore them. @YuxinShi0423 @jdidion